packages/core/ai/.prototools

```diff
@@ -0,0 +1 @@
+moon = "1.38.6"
```

Comment:
I don't think you need a separate version specification here. Lets remove this file and use one version for entire repo